src/api/firebase.js

```diff
@@ -91,3 +91,21 @@ export async function createNewList(listId) {
 	}
 	return response;
 }
+
+export async function checkIfListExists(listId) {
+	const listCollectionRef = collection(db, listId);
+
+	try {
+		const querySnapshot = await getDocs(listCollectionRef);
+
+		// If the collection exists and has documents, return the snapshot
+		if (!querySnapshot.empty) {
+			return querySnapshot;
+		} else {
+			// Collection doesn't exist
+			return null;
+		}
```

Comment:
**No need to change:** You don't need this `else` statement.

```js
if (condition) {
  return 'something'
}

// no need to wrap this in an `else` because the `return` statement above
// is "killing" the execution of the function.
return 'something else'
```